components/pricing_modal/downgrade_team_removal_modal/downgrade_team_removal_modal.test.tsx

```diff
@@ -0,0 +1,357 @@
+                        roles: 'team_user team_admin',
+                        delete_at: 0,
+                        scheme_guest: false,
+                        scheme_user: true,
+                        scheme_admin: true,
+                        explicit_roles: '',
+                    },
+                },
+                membersInTeam: {},
+                stats: {},
+                groupsAssociatedToTeam: {},
+                totalCount: 0,
+            },
+            admin: {},
+            general: {
+                license: {
+                    IsLicensed: 'true',
+                    Cloud: 'true',
+                },
+                credentials: {
+                    url: 'http://example.com',
+                },
+            },
+            cloud: {
+                subscription: {
+                    is_free_trial: 'false',
+                    trial_end_at: 0,
+                    product_id: 'prod_starter',
+                },
+                limits: {
+                    limitsLoaded: true,
+                    limits: {
+                        integrations: {
+                            enabled: 5,
+                        },
+                        messages: {
+                            history: 10000,
+                        },
+                        files: {
+                            total_storage: 10 * FileSizes.Gigabyte,
+                        },
+                        teams: {
+                            active: 1,
+                        },
+                        boards: {
+                            cards: 500,
+                            views: 5,
+                        },
+                    },
+                },
+                products: {
+                    prod_starter: {
+                        id: 'prod_starter',
+                        name: 'Cloud Starter',
+                        sku: CloudProducts.STARTER,
+                    },
+                    prod_enterprise: {
+                        id: 'prod_enterprise',
+                        name: 'Cloud Enterprise',
+                        sku: CloudProducts.ENTERPRISE,
+                    },
+                },
+            },
+        },
+        views: {
+            modals: {
+                modalState: {
+
+                    cloud_downgrade_choose_team: {
+                        open: 'true',
+                    },
+                },
+            },
+        },
+    };
+
+    test('renders modal', () => {
+        const mockStore = configureStore([thunk]);
+        const store = mockStore(state);
+        renderWithIntl(
+            <redux.Provider store={store}>
+                <DowngradeTeamRemovalModal
+                    product_id={'prod_starter'}
+                    starterProductName={'Cloud Starter'}
+                />
+            </redux.Provider>,
+        );
+        expect(screen.getByText('Confirm Plan Downgrade')).toBeInTheDocument();
```

Comment:
This looks more "jesty", but 0/5 on this style vs without the expect. `screen.getByText` throws an error/fails the test if it does not exist: https://testing-library.com/docs/queries/about#types-of-queries . The more common case of needing to combine with a jest expect is when you are doing a `queryBy` for something that does not exist. Then you expect it `not.toBeInTheDocument()`